src/pages/Main/Direction/Direction.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import styles from '../MainPage.module.scss';
+import icon001 from '../../../assets/icons/icon_001.png';
+import icon002 from '../../../assets/icons/icon_002.png';
+import icon003 from '../../../assets/icons/icon_003.png';
+import icon004 from '../../../assets/icons/icon_004.png';
+import icon005 from '../../../assets/icons/icon_005.png';
+
+interface DirectionItemProps {
+  src: string;
+  alt: string;
+  title: string;
+}
+
+const directionItems: DirectionItemProps[] = [
+  { src: icon001, alt: 'icon', title: 'QA' },
+  { src: icon002, alt: 'icon', title: 'Продакт менджмент' },
+  { src: icon003, alt: 'icon', title: 'Дизайн UX/UI' },
+  { src: icon004, alt: 'icon', title: 'Бизнес анализ' },
+  { src: icon005, alt: 'icon', title: 'Python' },
+];
+export function Direction() {
+  return (
+    <section className={styles.sectionDark}>
+      <div className={styles.container}>
+        <div className={styles.direction}>
+          <div className={styles.title}>
+            <h2>Прожарь свои навыки.</h2>
+            <h2>Узнай слабые места.</h2>
+            <h2>Доведи себя до готовности к новой должности</h2>
+          </div>
+          <div className={styles.directionList}>
+            {directionItems.map((item, index) => (
+              // eslint-disable-next-line react/no-array-index-key
```

Comment:
Why you disable eslint for this rule?

Reply:
using array indexes is a bad idea so I added the uuid library

Reply:
`console.log(crypto.randomUUID());` 
We don't need to add uuid library because we already have native node js functionality. But using random crypto like this may have potential problems. We should add id's for each item outside. Because every time the component is rendered, we will set new IDs for the element using the random function, which contradicts the idea of ​​​​tracking changes to each element using a unique ID. Simply put, we may end up with an endless re-rendering situation. But we can use this approach if we are sure that we will render this array only once at the time the component is mounted.

Reply:
I should set the id and save them in component state? and use useffect?

Reply:
No. You can either use crypto.randomUUID() in the same way as you used the uuid library, if you are sure that these elements are immutable; or you can change directionItems by adding id=crypto.randomUUID() for each item in the component variable declaration.
First variant:
```
              <div key={crypto.randomUUID()} className={styles.directionItem}>
                <img src={item.src} alt={item.alt} />
                <h4>{item.title}</h4>
              </div>
```
Second variant (use crypto.randomUUID() or manual add ID's like 0,1,2 etc.):
```
const directionItems: DirectionItemProps[] = [
  { src: icon001, alt: 'icon', title: 'QA', id: crypto.randomUUID()},
  { src: icon002, alt: 'icon', title: 'Продакт менджмент', id: 1},
  { src: icon003, alt: 'icon', title: 'Дизайн UX/UI' },
  { src: icon004, alt: 'icon', title: 'Бизнес анализ' },
  { src: icon005, alt: 'icon', title: 'Python' },
];
```

Reply:
i got it. I will use 1 version